drupal/DrupalTests.py

```diff
@@ -172,7 +170,7 @@ def run_behat_tests(repo, branch, build, alias, buildtype, url, ssl_enabled, jun
           test_method = '-f progress -o std -f junit -o xml'
         else:
           test_method = '-f pretty -o std'
-        if run("bin/behat -v --tags=\"%s\" %s" % (test_tags, test_method)).failed:
+        if run("/var/www/%s_%s_%s/vendor/bin/behat --config=/var/www/%s_%s_%s/tests/behat/behat.yml -v --tags=\"%s\" %s" % (repo, branch, build, repo, branch, build, test_tags, test_method)).failed:
```

Comment:
I would prefer to see this `--config` path as a config.ini setting that *defaults* to `/var/www/%s_%s_%s/tests/behat/behat.yml`

Reply:
Thanks for the review. I've made the changes you suggested, so now this PR includes support for customisable behat config paths, and removes the need for two composer.json files in the project.